ibazel/BUILD

```diff
@@ -20,7 +20,11 @@ go_binary(
     importpath = "github.com/bazelbuild/bazel-watcher/ibazel",
     # Since we don't have any cgo dependencies, force build to be pure
     # golang. This will make multi-os compatibility *MUCH* easier.
-    pure = "on",
+    pure = "auto",
+    cgo = select({
+        "@io_bazel_rules_go//go/platform:darwin": True,
```

Comment:
Can you explain why darwin is special here?

Reply:
Only darwin *needs* to be `cgo` since it relies on the FSEvents library which is with cgo (only macOS uses FSEvents). My understanding is that the final binary needs to be aware of cgo vs pure of all of its dependencies. Is there a way for the final binary to include cgo dependencies without being cgo itself? 

(I'll leave a comment in the code also.)

Reply:
Whoops I was wrong in my understanding. We can remote this attr altogether and it works out just fine!